README

```diff
@@ -69,7 +71,7 @@ To simply get counts of specific queries in a plaintext, whitespace-seperated co
 
 BUGS ETC.
 
-For bugs, remarks, suggestions etc. please email the author at j.h.stehouwer(_AT-)uvt.nl.
+For bugs, remarks, suggestions etc. please email the author at [j.h.stehouwer@uvt.nl](mailto:j.h.stehouwer@uvt.nl).
```

Comment:
Hi, should these days be j.h.stehouwer(_AT-)gmail.com the uvt.nl one doesn't exist anymore.
Not having the @ sign in there is to reduce spam somewhat.

The whole codebase probably needs a revamp though, as I made it 10+ years ago.

Reply:
Hi, sure! I was just trying to enhance the doc to make it more friendly. I can fork it again and update your email there if you want. 
By the way, this project was extremely helpful for learning.